base/nullable.jl

```diff
@@ -61,6 +61,68 @@ end
 
 get(x::Nullable) = isnull(x) ? throw(NullException()) : x.value
 
+"""
+    unsafe_get(x)
+
+Return the value of `x` for [`Nullable`](:obj:`Nullable`) `x`; return `x` for
```

Comment:
does the auto conversion script translate them from the current rst-in-markdown hybrid to the new style? if so it won't hurt to add if this gets merged before the conversion